lib/net.js

```diff
@@ -256,6 +258,9 @@ function Socket(options) {
 
   // Used after `.destroy()`
   this[BYTES_READ] = 0;
+
+  this[kLastTimeout] = 0;
+  this[kPrevTimeout] = 0;
```

Comment:
Well, the current `msecs` for the socket is already available via `socket._idleTimeout` and `http` would already know what value it is setting, so I don't see why it needs to be done in `net` (or perhaps why anything additional needs to be stored on the socket itself).